cni/network/network.go

```diff
@@ -1254,16 +1251,12 @@ func (plugin *NetPlugin) Update(args *cniSkel.CmdArgs) error {
 	targetEpInfo := &network.EndpointInfo{}
 
 	// get the target routes that should replace existingEpInfo.Routes inside the network namespace
-	logger.Info("Going to collect target routes for from targetNetworkConfig",
-		zap.String("pod", k8sPodName),
-		zap.String("namespace", k8sNamespace))
 	if targetNetworkConfig.Routes != nil && len(targetNetworkConfig.Routes) > 0 {
 		for _, route := range targetNetworkConfig.Routes {
 			logger.Info("Adding route from routes to targetEpInfo", zap.Any("route", route))
```

Comment:
Can we enhance this line with some info from the deleted line above? 
```suggestion
			logger.Info("Adding route from targetNetworkConfig to targetEpInfo", zap.Any("route", route))
```